delfin/tests/unit/fake_data.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2020 The SODA Authors.
+# Copyright (c) 2011 X.commerce, a business unit of eBay Inc.
+# Copyright 2010 United States Government as represented by the
+# Administrator of the National Aeronautics and Space Administration.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from delfin.db.sqlalchemy import models
+
+
+def fake_storage_pool_create():
+    fake_storage_pools = [models.StoragePool(), models.StoragePool(),
+                          models.StoragePool()]
+    fake_storage_pools[0].id = "14155a1f-f053-4ccb-a846-ed67e4387428"
```

Comment:
Can't we do it in on staement fake_storage_pools= [ models.StoragePool( prop1,prop2,...), secnd_pool, thied_pool ]?

Reply:
In sqlalchemy/models.py, all the models doesn't have __init__() method so as the above mentioned way we can't initiate attributes. The models has to be rewritten in more pythonic way.

Reply:
as mentioned above, one statement declaration is not possible but we can use dictionary init and avoid each entry init. same is handled now